torchrl/trainers/helpers/trainers.py

```diff
@@ -3,11 +3,13 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-import argparse
-from argparse import ArgumentParser, Namespace
-from typing import Optional, Union
+from dataclasses import dataclass
+from typing import Optional, Union, List
 from warnings import warn
 
+from hydra import compose, initialize
```

Comment:
any way we can use the same logic as above to avoid importing that here? Using a try / except for instance?
Since hydra is an optional dependency, I would assume that something will break from time to time when users don't install it but try to import something from this module.

Reply:
https://github.com/BoboBananas/rl/pull/3/commits/3e4a58a2220b0301563250bae00dcb9f5543df79